casa.py

```diff
@@ -115,16 +118,17 @@ def casa_set_status(self, project_id, delegator_id, bug_resolution):
         """
         # Map decision bugzilla=>(casa decision label, casa decision)
         decision_map = {
-                         'FIXED': ('Completed: No oustanding issues found', 'done'),
-                         'INVALID': ('Pending', 'none'),
-                         'DUPLICATE': ('Pending', 'none'),
-                         'WONTFIX': ('Do not use', 'rejected'),
-                         'INCOMPLETE': ('Warning: Outstanding issues, refer to the RRA', 'done')
-                       }
+            'FIXED': ('Completed: No oustanding issues found', 'done'),
+            'INVALID': ('Pending', 'none'),
+            'DUPLICATE': ('Pending', 'none'),
+            'WONTFIX': ('Do not use', 'rejected'),
+            'INCOMPLETE': ('Warning: Outstanding issues, refer to the RRA', 'done')
+        }
         decisionLabel, decision = decision_map.get(bug_resolution)
 
         # Set it
-        payload = {'delegatorId': delegator_id, 'decision': decision, 'decisionLabel': decisionLabel }
+        payload = {'delegatorId': delegator_id,
```

Comment:
please use 120col not 80 ;-)

Reply:
I think this was either my editor or me wanting to make it look nice, I can have a second look soonish.